script/build.sh

```diff
@@ -6,5 +6,5 @@ rm -rf dist/
 ./script/bundle.sh test Test.Test
```

Comment:
I feel like we should probably inline `./script/bundle.sh` here and just call `./script/util/clean.sh` and `./script/util/bundle.sh` directly. (I think this is the only use of `./script/bundle.sh`.)